docs/migration.md

```diff
@@ -251,6 +226,97 @@ consider additional positioning prop support on a case-by-case basis.
 - Migrate to `@zendeskgarden/container-utilities` and `@zendeskgarden/react-theming` to continue
   receiving updates.
 
+### Deprecated components
+
+The following subcomponents have been renamed to streamline imports and improve
+affinity with their parent components.
```

Comment:
[nit] To avoid repetitions, should we move the migration instructions to the intro?

```suggestion
The following subcomponents have been renamed to streamline imports and improve
affinity with their parent components. Subcomponent exports have been deprecated and will be removed in a future major version. Update to subcomponent properties:
```

Reply:
Yeah, good call.

Reply:
@ze-flo updated in [833bea6](https://github.com/zendeskgarden/react-components/pull/1895/commits/833bea63560b6e77b2af87506c329d9a708b66c4)